From: "Steinar H. Gunderson" <sesse@debian.org>
Date: Sat, 23 Feb 2019 14:01:23 +0100
Subject: Change PREFIX to /usr

---
 nageru/flags.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nageru/flags.h b/nageru/flags.h
index 4d990f8..3594697 100644
--- a/nageru/flags.h
+++ b/nageru/flags.h
@@ -18,7 +18,7 @@ struct Flags {
 	bool uncompressed_video_to_http = false;
 	bool x264_video_to_http = false;
 	bool x264_video_to_disk = false;  // Disables Quick Sync entirely. Implies x264_video_to_http == true.
-	std::vector<std::string> theme_dirs { ".", "/usr/local/share/nageru" };
+	std::vector<std::string> theme_dirs { ".", "/usr/share/nageru" };
 	std::string recording_dir = ".";
 	std::string theme_filename = "theme.lua";
 	bool locut_enabled = true;
